webcurator-ui/src/stores/targetList.ts

```diff
@@ -33,4 +35,96 @@ export const useTargetListFiltertore = defineStore('TargetListFilterStore', () =
         selectedUser, 
         selectedState, 
     }
+})
+
+export const useTargetListDataStore = defineStore('TargetListDataStore', () => {
+    const userProfile = useUserProfileStore();
+    const users = useUsersStore();
+    const targetList = ref([]);
+    const loadingTargetList = ref(false);
+    const filteredTargetList = ref();
+    const searchTerms = useTargetListSearchStore();
+    const filters = useTargetListFiltertore();
+    const rest: UseFetchApis = useFetch();
+
+    const filter = () => {
+        const ret = [];
+        for (let idx = 0; idx < targetList.value.length; idx++) {
+            const target: any = targetList.value[idx];
+            if (filters.selectedAgency.code !== "" && target.agency !== filters.selectedAgency.name) {
+                continue;
+            }
+
+            if (filters.selectedUser.code !== "" && target.owner !== filters.selectedUser.code) {
+                continue;
+            }
+
+            let isInSelectedStates = false;
+            for (const idx in filters.selectedState) {
+            const stateOption: any = filters.selectedState[idx];
+            if (target.state === stateOption.code) {
+                isInSelectedStates = true;
+                break;
+            }
+            }
+            if (filters.selectedState.length > 0 && !isInSelectedStates) {
+            continue;
```

Comment:
A bit of inconsistent indentation in this file :)

Reply:
I've formatted the indentation. Thanks for the feedback, ~